depenguinme.sh

```diff
@@ -247,50 +250,45 @@ wget -qc -O "${MFSBSDFILE}" "${MFSBSDISO}" || exit_error "Could not download mfs
 
 # check for drives
 echo "Searching sd[abcd] and nvme"
+
 set +e
 
-# function to check for all valid drive names and return just the name
-_get_all_valid_drives() {
-	# Capture the complete output of lsblk for debugging
-	lsblk_output=$(lsblk -no NAME,TYPE,TRAN)
-
-        # Count the number of NVMe drives (doesn't work with hetzner)
-        nvme_count=$(echo "$lsblk_output" | grep -c '^nvme')
-
-        # If there are 1 or more NVMe drives, list only those
-        if [ "$nvme_count" -ne 0 ]; then
-		USENVME="$nvme_count"
-                echo "$lsblk_output" | awk '/^nvme/ && $2 == "disk" {print $1}'
-        else
-        # Otherwise, list all sd* drives, excluding CD-ROM and USB drives
-		unset USENVME
-                echo "$lsblk_output" | awk '$2 == "disk" && $3 != "usb" && $3 != "sr" && $1 ~ /^sd/ {print $1}'
-        fi
-}
+# Capture the complete output of lsblk for debugging
```

Comment:
removed the function, as it was only called right after it was declared (and none of its variables were local).